crates/autopilot/src/limit_order_quoter.rs

```diff
@@ -57,63 +58,70 @@ impl LimitOrderQuoter {
             if orders.is_empty() {
                 break;
             }
-            for order in orders {
-                match self
-                    .quoter
-                    .calculate_quote(QuoteParameters {
-                        sell_token: order.data.sell_token,
-                        buy_token: order.data.buy_token,
-                        side: match order.data.kind {
-                            OrderKind::Buy => OrderQuoteSide::Buy {
-                                buy_amount_after_fee: order.data.buy_amount,
-                            },
-                            OrderKind::Sell => OrderQuoteSide::Sell {
-                                sell_amount: SellAmount::BeforeFee {
-                                    value: order.data.sell_amount + order.data.fee_amount,
+            for chunk in orders.chunks(10) {
```

Comment:
At some point if we notice we need to adjust this number, we should make it configurable instead of being a constant. I think just picking a "good" number for now makes sense though.